### ISSUE TITLE
indentation rule is conflict with prettier auto format

### ISSUE THREAD

**Issue by copiali** [stylelint/stylelint#6445]
### What steps are needed to reproduce the bug?

I have auto format on save in my vscode and if it exceed the expected length, prettier will auto format it like below:
![image](https://user-images.githubusercontent.com/5318202/199406050-6273cf83-14bb-40e0-91eb-3a2fcafe461a.png)

It will indent line 127 and not indent line 128.

### What Stylelint configuration is needed to reproduce the bug?

```json
{
    "extends": [
        "stylelint-config-standard-scss",
        "stylelint-config-prettier-scss"
    ],
    "rules": {
        "indentation": 4,
    }
}
```

### How did you run Stylelint?

CLI with `stylelint "src/**/*.scss"`

### Which version of Stylelint are you using?

`14.14.0`

### What did you expect to happen?

No problems to be reported.

### What actually happened?

The following problems are reported:

```
 127:5  ✖  Expected indentation of 0 spaces  indentation
 138:9  ✖  Expected indentation of 4 spaces  indentation
```

### Proposal to fix the bug

I've tried some different options for this rule but seems none of them can fix my issue here.

**Comment by ybiquitous** [comment]
@copiali Thanks for using the template.

`stylelint-config-prettier-scss` (`stylelint-config-prettier`) turns off the `indentation` rule,

- https://github.com/prettier/stylelint-config-prettier-scss/blob/b294d1bbea8cbb640b759b82abe0ce667316a7f1/src/index.js#L4
- https://github.com/prettier/stylelint-config-prettier/blob/d922cd2f9b97b1fc8e7feabdc49e6310acfa3d95/src/index.js#L5

so I recommend removing the `indentation: 4` settings from your config and relying on Prettier for formatting. Or is there a reason to turn on the rule?

**Comment by copiali** [comment]
> @copiali Thanks for using the template.
> 
> `stylelint-config-prettier-scss` (`stylelint-config-prettier`) turns off the `indentation` rule,
> 
> * https://github.com/prettier/stylelint-config-prettier-scss/blob/b294d1bbea8cbb640b759b82abe0ce667316a7f1/src/index.js#L4
> * https://github.com/prettier/stylelint-config-prettier/blob/d922cd2f9b97b1fc8e7feabdc49e6310acfa3d95/src/index.js#L5
> 
> so I recommend removing the `indentation: 4` settings from your config and relying on Prettier for formatting. Or is there a reason to turn on the rule?

@ybiquitous thanks for your reply.
My concern for removing the indentation rule is for some devs if they are not using vscode editor or for some reason, they skip the auto format from prettier. The build will succeed for the incorrect indentation as there's no rule in the stylelint config. But I want to capture this incorrect format and fail the build in this case.

**Comment by ybiquitous** [comment]
> I want to capture this incorrect format and fail the build in this case.

Can you use [`prettier --check`](https://prettier.io/docs/en/cli.html#--check) in your build process (e.g. CI) instead of turning on the `indentation` rule of Stylelint?

We've [frozen stylistic rules](https://stylelint.io/user-guide/rules#enforce-stylistic-conventions) (including `indentation`) and recommended using a pretty printer like Prettier instead. If there is a bug in such rules, we won't fix it.

**Comment by copiali** [comment]
@ybiquitous thanks for your suggestion. Added prettier --write in pre-commit hook and it works well!
